.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
this file should not be added to git, please remove it and add `.vscode` to the .gitignore